client/hostinfo_unix.cpp

```diff
@@ -1238,6 +1245,9 @@ int HOST_INFO::get_cpu_info() {
     strlcpy( p_model, cpuInfo.name.fromID, sizeof(p_model));
 #elif defined(__HAIKU__)
     get_cpu_info_haiku(*this);
+#elif WASM
+    strlcpy( p_vendor, "WASM", sizeof(p_vendor));
+    strlcpy( p_model, "Linux", sizeof(p_model));
```

Comment:
Technically this could be run on any OS, so just hardcode 'Linux' is not nice.

Reply:
This is hardcoded until it will be running and we understand how to ask os type in JavaScript. 
Not immediate change.

Reply:
Then hardcode it to `WASM` as well. It's better than report `linux` on `Windows`